### PR TITLE
Fix `torch.compiler.cudagraph_mark_step_begin` example

### PR DESCRIPTION
**Pull request by carmocca**
Per title

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

### CODE REVIEW

File: torch/compiler/__init__.py [hunk 1]
```diff
@@ -109,12 +109,12 @@ def cudagraph_mark_step_begin():
...
+        for _ in range(5):
+            torch.compiler.cudagraph_mark_step_begin()
             rand_foo() + rand_foo()
 
     For more details, see `torch.compiler_cudagraph_trees <https://pytorch.org/docs/main/torch.compiler_cudagraph_trees.html>`__
     """
-    import torch._inductor
+    from torch._inductor import cudagraph trees
 
-    torch._inductor.cudagraph_trees.mark_step_begin()
```
Comment (carmocca) [inline torch/compiler/__init__.py:120]: This was raising

```python
Traceback (most recent call last):
  File "/home/carlos/kk.py", line 8, in <module>
    torch.compiler.cudagraph_mark_step_begin()
  File "/home/carlos/nightly-env/lib/python3.10/site-packages/torch/compiler/__init__.py", line 120, in cudagraph_mark_step_begin
    torch._inductor.cudagraph_trees.mark_step_begin()
AttributeError: module 'torch._inductor' has no attribute 'cudagraph_trees'
```

**Review by eellison — APPROVED** [review]
thanks, sorry about that! 

Would you mind changing this usage here to the official api? https://github.com/pytorch/pytorch/blob/main/test/inductor/test_cudagraph_trees.py#L1276

would have caught this typo 🤦

**Comment by eellison** [comment]
hey, looks like you need to fix lint and then we can land `pip install lintrunner; lintrunner init; lintrunner -a`

**Pull request closed without merging**